### ISSUE TITLE
Unabled to Download CloudPilot Trial

### ISSUE THREAD

**Issue by castaoh** [MicrosoftDocs/visualstudio-docs#5496]
I wasn't able to download the free trial. When I clicked the link it requested access to AAD that an admin needs to provide. Raised this to the publisher, but no response. Also a couple of customers with VSO Enterprise MSDN or Cloud Subscription have the benefit available on https://my.visualstudio.com/benefits

**Comment by WilliamAntonRohm** [comment]
cc @evanwindom for awareness

**Comment by evanwindom** [comment]
@WilliamAntonRohm  - Please assign these to me, not to Caity.

**Comment by evanwindom** [comment]
@castaoh - Thanks for bringing this issue to our attention.  What link are you clicking to download the trial?

**Comment by WilliamAntonRohm** [comment]
@evanwindom -- thank you for clarifying.

**Comment by castaoh** [comment]
> @castaoh - Thanks for bringing this issue to our attention. What link are you clicking to download the trial?

According to this link https://docs.microsoft.com/en-us/visualstudio/subscriptions/vs-cloudpilot 

1 Includes: Not for Resale (NFR), FTE, Most Valuable Professional (MVP), Regional Director (RD), Microsoft Partner Network (MPN), Visual Studio Industry Partner (VSIP), Microsoft Certified Trainer, BizSpark, Imagine

But FTE VSO Subscriptions in the field aren´t available to get the benefit, for this reason I tried to download the trial using https://appsource.microsoft.com/en-us/product/web-apps/cloudatlasinc.36d534d9-ab47-4cd8-93d3-2be7df682782?tab=Overview

Using trial button on above page is when login issue arise.

**Comment by castaoh** [comment]
According to this link https://docs.microsoft.com/en-us/visualstudio/subscriptions/vs-cloudpilot

1 Includes: Not for Resale (NFR), FTE, Most Valuable Professional (MVP), Regional Director (RD), Microsoft Partner Network (MPN), Visual Studio Industry Partner (VSIP), Microsoft Certified Trainer, BizSpark, Imagine

But FTE VSO Subscriptions in the field aren´t available to get the benefit, for this reason I tried to download the trial using https://appsource.microsoft.com/en-us/product/web-apps/cloudatlasinc.36d534d9-ab47-4cd8-93d3-2be7df682782?tab=Overview

Using trial button on above page is when login issue arise.

Thanks in advance...

Regards.

________________________________
De: evanwindom <notifications@github.com>
Enviado: miércoles, 17 de junio de 2020 6:19 p. m.
Para: MicrosoftDocs/visualstudio-docs <visualstudio-docs@noreply.github.com>
Cc: castaoh <oskarcastaneda@hotmail.com>; Mention <mention@noreply.github.com>
Asunto: Re: [MicrosoftDocs/visualstudio-docs] Unabled to Download CloudPilot Trial (#5496)


@castaoh<https://github.com/castaoh> - Thanks for bringing this issue to our attention. What link are you clicking to download the trial?

**Comment by evanwindom** [comment]
@castaoh - Thank you for the additional context.  I walked through the process, and got the same result I believe you were talking about.  It looks like permission from an Azure Active Directory administrator within @microsoft.com is required.  Unfortunately, AppSource is unrelated to the offers provided through the Visual Studio subscriptions, so I don't have any solutions to offer you from our side.    

I looked at the Help and Support links on the AppSource product page for CloudPilot, and they both link to resources at CloudAtlas/UnifyCloud.  If you've submitted a support request there, you've already done what I would suggest is the best course of action.  If you find a solution, I would appreciate hearing about it so we'll have better guidance to provide other FTEs.
